zkevm-circuits/src/evm_circuit/util/common_gadget/tx_eip1559.rs

```diff
@@ -90,6 +103,18 @@ impl<F: Field> TxEip1559Gadget<F> {
             let gas_fee_cap_lt_base_fee =
                 LtWordGadget::construct(cb, &gas_fee_cap, &base_fee);
 
+            // calculating min(
+            //     gas_tip_cap
+            //     gas_fee_cap - base_fee_per_gas,
+            // );
+            let diff_gas_base_fee = cb.query_word_rlc();
+            let gas_sub_base_fee = AddWordsGadget::construct(cb, [base_fee.clone(), diff_gas_base_fee.clone()], gas_fee_cap.clone());
+            let lt_gas_tip_gas_fee_sub_base_fee = LtWordGadget::construct(cb, &gas_tip_cap, &diff_gas_base_fee);
+            // let effective_gas_price = priority_fee_per_gas + base_fee_per_gas;
+            let priority_fee_per_gas = cb.query_word_rlc();
```

Comment:
is this priority_fee_per_gas constrained anywhere?

Reply:
constraint in local changes,  pushed.